structured_data/2021_02_26_scikit_learn_pipelines/README.md

```diff
@@ -0,0 +1,9 @@
+# Tabular data: Scikit-learn Pipelines and LightGBM
+
+When working on a data science problem you are faced with multiple challenges. How to preprocess my data in a structured way? How to avoid dataset leakage? How to efficiently do hyper parameter tuning?
```

Comment:
```suggestion
When working on a data science problem you are faced with multiple challenges. How do you preprocess your data in a structured way? How to avoid dataset leakage? How to efficiently do hyper parameter tuning?
```